### ISSUE TITLE
[BUG] - Traefik rate limited by Let's Encrypt after several pod restarts

### ISSUE THREAD

**Issue by sblair-metrostar** [nebari-dev/nebari#2174]
### Describe the bug

[Traefik documentation](https://doc.traefik.io/traefik/https/acme/) warns to persist the acme.json certificate storage to avoid being rate limited by Let's Encrypt, but Nebari's deployment isn't configuring that. If the traefik pod is restarted enough times within a given week without this setup, Let's Encrypt will rate limit the certificate issue request and cause the site to default back to self-signed.

This has happened to me [many, many times](https://github.com/nebari-dev/nebari/issues/1581#issuecomment-1335570056) when I'm developing or testing changes to the platform or dealing with conda storage related node crashes, usually requiring a workaround involving installation of cert-manager and switching to zerossl or another cert issuing service until the Let's Encrypt backoff period elapses.

### Expected behavior

traefik pod restarts should not result in new certificates being issued when configured for lets-encrypt.

Options:
1. Configure a persistent volume mount for the [acme.json](https://github.com/nebari-dev/nebari/blob/d612e9b5f63db0e810c1fe6fc0cd61b5bfa62641/src/_nebari/stages/kubernetes_ingress/template/modules/kubernetes/ingress/main.tf#L12) storage path.
2. Replace Traefik's native Let's Encrypt integration with cert-manager for more issuer options and improved scalability.

### OS and architecture in which you are running Nebari

Linux, x64

### How to Reproduce the problem?

1. Configure the Nebari certificate with lets-encrypt

``` yaml

certificate:
  type: lets-encrypt
  acme_email: notreal@example.com
  acme_server: https://acme-v02.api.letsencrypt.org/directory

```

2. Delete the traefik pod 5 or more times in 1 week.

### Versions and dependencies used.

Nebari 2023.11.1

### Compute environment

AWS

**Comment by Princess4ogb** [comment]
I am working on this ticket, Please find the gitlab issue as below.
https://gitlab.jatic.net/jatic/team-metrostar/t-e-platform/-/issues/345

**Comment by kcpevey** [comment]
Resolved by #2352
